### ISSUE TITLE
Add disk reads/writes to OS X and Windows

### ISSUE THREAD

**Issue by dbwiddis** [dblock/oshi#192]
#147 added disk statistics, using Linux's sysfs (accessed via udev commands/structure) which included reads/writes.  At the time, I couldn't find an easy way to include this information for other OS's.  However, with a bit more digging I've found some more leads.

OS X has the `iostat` command which shows only transfers, but [digging into the source code](http://opensource.apple.com//source/system_cmds/system_cmds-279.6/iostat.tproj/iostat.c) I can see that the information is pulled from the registry where a search for `IOBlockStorageDriver` uncovers a `Statistics` field with a dictionary containing the read and write values.   It looks like we can start with the BSD Name of the drive (disk0, disk1, etc.) to find it in the registry.  The parent of the matching bsdname entry is the `IOBlockStorageDriver` with the statistics.

Windows is a tougher nut to crack.  There is a [command line fsutil](https://technet.microsoft.com/en-us/library/cc788080(v=ws.11).aspx) with the information, but it requires an elevated command prompt (admin priveleges) to get the info: `fsutil fsinfo statistics <drive letter>`   I don't think we can rely on elevated privileges (especially on Windows) but it gives me hope that the information is somewhere in the system that we can get to it.

Windows also has PerfMon counters for disk I/O but they're read/write per second (so we'd have to multiply by elapsed time and maintain a cumulative counter each time the method was called, similar to what we do for CPU ticks) but they would start at zero when OSHI starts.  I still think this is a better option than leaving it at zero, however, so I'll keep digging but use this as a fall back.

**Comment by dbwiddis** [comment]
OK, this is too easy.  Having earlier investigated the PerfMon counters, I completely ignored the [Win32_PerfRawData_PerfDisk_PhysicalDisk](https://msdn.microsoft.com/en-us/library/aa394308(v=vs.85).aspx) WMI class because the names were similar.  However, trying out some queries, it appears that one can ignore the "per second" bit... the WMI counters report cumulatively increasing integer numbers.... (and a timestamp) so I can report these values straight out.  PerfMon (which was harder to do to begin with) does the math with these numbers and the timestamp...  so it actually looks like the Windows result is trivial and the OS X result is less trivial but doable.

**Comment by dbwiddis** [comment]
```
C:\>wmic path Win32_PerfRawData_PerfDisk_PhysicalDisk 
    get Name,DiskWriteBytesPerSec,DiskReadBytesPerSec,Timestamp_PerfTime
DiskReadBytesPersec  DiskWriteBytesPersec  Name    Timestamp_PerfTime
7017398784           5875023872            0 C:    4576877479168
7017398784           5875023872            _Total  4576877479168
```

**Comment by dbwiddis** [comment]
Windows done.  That was too easy.  (And yes the docs were clear that the "raw data" was the basis for the "formatted data" which the counters use.)

Disks:
 \\.\PHYSICALDRIVE0: (model: Windows 7-0 ATA Device (Standard disk drives) - S/N: 1ACA4T5WH3MDEFMY8AJ4) size: 68.7 GB, **reads: 6.6 GiB, writes: 5.5 GiB**

**Comment by dbwiddis** [comment]
And after a long, hard-fought battle with Pointers that I eventually won, OS X is done...
```
Disks:
 disk0: (model: SanDisk Ultra II 960GB - S/N: 161008800550) size: 960.2 GB, reads: 1.5 TiB, writes: 903.8 GiB 
 disk4: (model: Disk Image - S/N: ) size: 960.0 GB, reads: 2.2 GiB, writes: 568.2 MiB 
 disk2: (model: Disk Image - S/N: ) size: 5.8 TB, reads: 2.6 GiB, writes: 47.1 GiB 
 disk3: (model: Disk Image - S/N: ) size: 26.7 MB, reads: 14.8 MiB, writes: 0 bytes 
```
I'll clean things up and commit tomorrow.

**Comment by dbwiddis** [comment]
Fixed in #193
